egs/swbd/s5c/local/reverberate/IR_stats.m

```diff
@@ -0,0 +1,348 @@
+function [rt,drr,cte,cfs,edt] = IR_stats(x, fs, varargin)
+%function [rt,drr,cte,cfs,edt] = IR_stats(filename, varargin)
```

Comment:
This script is not central to the recipe. It would just be necessary for
generating the metadata for files being uploaded to openslr.org.

On Wed, Apr 6, 2016 at 10:55 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/reverberate/IR_stats.m
> <https://github.com/kaldi-asr/kaldi/pull/667#discussion_r58812975>:
>
> > @@ -0,0 +1,348 @@
> > +function [rt,drr,cte,cfs,edt] = IR_stats(x, fs, varargin)
> > +%function [rt,drr,cte,cfs,edt] = IR_stats(filename, varargin)
>
> I see this file seems to be available online at
>
> http://www.mathworks.com/matlabcentral/fileexchange/42566-impulse-response-acoustic-information-calculator/content/IR_stats.m
> This could be a download, I guess- or a patched download if needed.
> Anyway, at that location, it says the code is covered by the BSD license,
> which implies that we can redistribute it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/667/files/1683e258c9f571271668ab6e1d2b0898e7b7ed33#r58812975>
>